DESCRIPTION

```diff
@@ -2,24 +2,24 @@ Package: bank
 Title: Extra Cache
 Version: 0.0.0.9002
 Authors@R: 
-    person(given = "Colin",
-           family = "Fay",
-           role = c("aut", "cre"),
-           email = "contact@colinfay.me",
+    person("Colin", "Fay", , "contact@colinfay.me", role = c("aut", "cre"),
            comment = c(ORCID = "0000-0001-7343-1846"))
 Description: More cache backends.
 License: MIT + file LICENSE
-Encoding: UTF-8
-LazyData: true
-Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.1.2
 Imports: 
     attempt,
-    R6 (>= 2.5.0)
-Suggests: 
+    DBI,
+    digest,
     mongolite (>= 2.2.0),
```

Comment:
These packages were put inside `Suggests` instead of `Imports` on purpose, can you put them back there? Thanks :)